tools/StaticAnalysis/Exceptions/AzureRM.Compute/BreakingChangeIssues.csv

```diff
@@ -21,3 +21,82 @@
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMDataDiskCommand","Add-AzureRmVMDataDisk","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachine' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity' to 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachineIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMNetworkInterfaceCommand","Add-AzureRmVMNetworkInterface","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachine' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity' to 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachineIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.AddAzureVMNetworkInterfaceCommand","Add-AzureRmVMNetworkInterface","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachine' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity' to 'Microsoft.Azure.Commands.Compute.Models.PSVirtualMachineIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.AddAzureRmVmssAdditionalUnattendContentCommand","Add-AzureRmVmssAdditionalUnattendContent","0","3000","The type of property 'Identity' of type 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSet' has changed from 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity' to 'Microsoft.Azure.Commands.Compute.Automation.Models.PSVirtualMachineScaleSetIdentity'.","Change the type of property 'Identity' back to 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIdentity'."
+"Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.AddAzureRmVmssAdditionalUnattendContentCommand","Add-AzureRmVmssAdditionalUnattendContent","0","3000","The type of property 'PrivateIPAddressVersion' of type 'Microsoft.Azure.Management.Compute.Models.VirtualMachineScaleSetIPConfiguration' has changed from 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.IPVersion]' to 'System.String'.","Change the type of property 'PrivateIPAddressVersion' back to 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.IPVersion]'."
```

Comment:
Just a comment. This was discussed offline. It is an enum becoming a string which is not a breaking change.